cmd/cli/main.go

```diff
@@ -0,0 +1,17 @@
+package main
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/sirupsen/logrus"
+	firestore "pantheon.io/edrt-policy-docs-functions/internal/store"
+)
+
+func main() {
+	fmt.Println("This will be a CLI allowing interactive testing of logic in this repo during development.")
+	fmt.Println("Will probably use a library like cobra to set up the CLI.")
+	f := firestore.New(logrus.New(), "rhamilton-001")
```

Comment:
In the code, I'd use viper to pull them in

Setting environment variables will need to be done in the cloud function definition in gce-terraform. This kind of annoys me since they won't live with the code.

Another option is to create a yaml file in the repo that we read in with viper. The easy way to do this is with viper's [AddConfigPath](https://github.com/spf13/viper/blob/v1.15.0/viper.go#L564) function, but that relies on the file system and I'm not certain about how that will function in a cloud function. The second option is to load the yaml file ourselves, parse it into a `map[string]interface{}` and use [MergeConfigMap](https://github.com/spf13/viper/blob/v1.15.0/viper.go#L1604) (or one of the similar functions). This would almost certainly work because we'd be able to embed the yaml file in the binary and it wouldn't depend on the file system.

Reply:
Thanks for the detailed explanation. Implementation details aside, I like the idea of having a yaml file living in the repo rather than over in gce-terraform with the definition. I have found references to `AddConfigPath` in other cloud functions (cursory google search), so I suspect we'll be ok with that, but we can kick the implementation can down the road some.

Reply:
I've added a `config.yaml` file in a directory called `config`. We will probably want to split this out to prod and non-prod and have circleci grab the correct one before deploying.

I know the standard is to have a `devops` folder. I originally did that, but push-trolley does not deploy the devops folder, so the config would not be deployed. Happy to discuss alternatives here.